src/SkillService.Mappers/Models/ShortSkillInfoMapper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using LT.DigitalOffice.SkillService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.SkillService.Models.Db;
+using LT.DigitalOffice.SkillService.Models.Dto.Models;
+
+namespace LT.DigitalOffice.SkillService.Mappers.Models
+{
+  public class ShortSkillInfoMapper : IShortSkillInfoMapper
+  {
+    public ShortSkillInfo Map(DbSkill dbSkill)
+    {
+      if (dbSkill is null)
```

Comment:
Maybe `return dbSkill is null ? null : new ShortSkillInfo{...};`

Reply:
Yep